### ISSUE TITLE
Googleapi error 403 Required 'compute.zones.list' permission for 'projects/XXX', forbidden

### ISSUE THREAD

**Issue by Golge** [terraform-google-modules/terraform-google-vault#206]
### TL;DR

vault >> modules >> cluster >> main.tf
```
data "google_compute_zones" "available" {
  project = var.project_id
  region  = var.region
}

```

### Observed behavior

│ Error: googleapi: Error 403: Required 'compute.zones.list' permission for 'projects/XXX', forbidden
│ 
│   with module.cluster.data.google_compute_zones.available,
│   on modules/cluster/main.tf line 41, in data "google_compute_zones" "available":
│   41: data "google_compute_zones" "available" {

### Terraform Configuration

```hcl
project_id = "XXX"
region = "europe-west4"
storage_bucket_class = "REGIONAL"
storage_bucket_force_destroy = "true" # SET FALSE FOR PRODUCTION
storage_bucket_location = "EUROPE-WEST4"
vault_machine_type = "n1-standard-1"
vault_max_num_servers = "3"
vault_min_num_servers = "2"
vault_version = "1.16.2" # LATEST VERSION
kms_keyring = "vault-keyring-01"
kms_crypto_key = "vault-key-01"
```

### Terraform Version

```sh
OpenTofu v1.7.1
```

### Additional information

I have my permissions but still giving the error 403
![tofu](https://github.com/terraform-google-modules/terraform-google-vault/assets/10224996/b85be6cd-9b47-4a99-839b-dd1a79ef3642)
